crates/vhost-user-backend/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Changed
 - Change uses of master/slave for frontend/backend in the codebase.
+- BREAKING: Converted generic type parameters of VhostUserBackend into
```

Comment:
We usually put the link to the PR with the title, but I don't know why we changed with the other merged PR.

I'd rather still have the link, but we can add it later when we prepare the release.

Reply:
I guess this also means a major version jump?

Reply:
> I guess this also means a major version jump?

Yes, but the other change - which renamed types - already requires that anyway. Thats why it is a good idea to batch these two changes together :)

Reply:
> I guess this also means a major version jump?

Yep! Since we still are on `0.y.z`, increasing `y` is a major version jump: https://doc.rust-lang.org/cargo/reference/resolver.html#semver-compatibility

Reply:
> We usually put the link to the PR with the title, but I don't know why we changed with the other merged PR.
> 
> I'd rather still have the link, but we can add it later when we prepare the release.

added. Also dropped the `BREAKING:` since the other change - which also is breaking in a similar way - does not have it.